docs/Makefile

```diff
@@ -1,12 +1,12 @@
 # Minimal makefile for Sphinx documentation
 #
 
-# You can set these variables from the command line.
-SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
-SPHINXPROJ    = DjangoCorrelationId
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = .
-BUILDDIR      = ../build/sphinx
+BUILDDIR      = _build
```

Comment:
Isn't there a conflict with `exclude_patterns` which includes the new BUILDDIR path ?
https://github.com/Polyconseil/django-cid/pull/62/commits/419cbda0204d0f6ba5177df5449e45c09ab564c1#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R45

Reply:
If I remember correctly, this is the default Sphinx configuration. The Makefile says that we build in a `_build/` directory, and the Sphinx configuration says that Sphinx should not look for source files in that directory. It seems ok to me. Did I miss something?

Reply:
Okay, I misunderstood the behaviour of `exclude_patterns`. All good for me :+1: